sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsTests.cs

```diff
@@ -1260,6 +1261,45 @@ public async Task CreateChatThreadAsyncShouldExposePartialErrors()
             Assert.AreEqual("19:e5e7a3fa5f314a01b2d12c6c7b37f433@thread.v2", createChatThreadResult.ChatThread.Id);
         }
 
+        [Test]
+        public void CreateChatThreadWithThreadRetentionPolicyShouldSucceed()
```

Comment:
Could you please add tests for the UpdateThread? Especially for the coverage of different combinations of properties update, like only update topic, update retention policy to null/not null, update metadata, update both topic and metadata, update both topic and retention policy.